doc/userguide/manpages/suricatactlfilestore.rst

```diff
@@ -0,0 +1,51 @@
+Suricata Control Filestore
+==========================
+
+SYNOPSIS
+--------
+
+**suricatactl filestore** [-h] <command> [<args>]
+
+DESCRIPTION
+-----------
+
+**Filestore** allows to store files to disk if a particular signature is matched.
+
+Syntax for usage in suricata.yaml:
+
+**filestore:<direction>,<scope>;**
```

Comment:
this looks like rule syntax. Pretty sure it doesn't belong here.

Reply:
Oops. I was supposed to add both suricata.yaml and rule configuration for filestore. Fixing this. Do you think it should be there at all? Does it make sense according to you to have a manpage dedicated to filestore?

Reply:
I think so, but this PR seems to introduce 2 of them? suricatactlfilestore.rst and suricatactl-filestore.rst? The latter one looks better.

Reply:
Actually, this is following the sub-command manpages as by `git-submodules`. They have one for `git-submodules` where they write about the command usage and options and then there is one `gitsubmodules` where they explain what git submodules actually are.
Since `suricatactl` is mostly a WIP and more commands shall be added in future, each command will have their separate man page. Do youthink I should still keep just the latter?

Reply:
Ah interesting, didn't know what. I think for our purpose we can do without the explanation of what filestore is, and only do the page that explains the usage of the command.